plugins/generic/pln/PLNPlugin.inc.php

```diff
@@ -19,9 +19,7 @@
 
 define('PLN_PLUGIN_NAME','plnplugin');
 
-define('PLN_PLUGIN_NETWORKS', serialize(array(
-	'PKP' => 'pkp-pln.lib.sfu.ca'
-)));
+define('PLN_PLUGIN_NETWORK', 'pkp-pln.lib.sfu.ca');
```

Comment:
Why not include the http:// here? It looks (at a glance) like all uses of this constant prepend it anyway.